samples/src/Microsoft.Azure.IIoT.App/src/Pages/Discoverers.razor

```diff
@@ -76,14 +76,15 @@
                     }
                 </td>
                 <td>
-                    @{ string ScanMask = "--"; }
-                    @{ string PortRange = "--";}
-                    @{ string MaxNetworkProbes = "--";}
-                    @{ string MaxPortProbes = "--";}
-                    @{ string NetworkProbeTimeoutMs = "--";}
-                    @{ string PortProbeTimeoutMs = "--";}
-                    @{ string IdleTimeBetweenScansSec = "--";}
-                    @{ string ActivationFilter = "--";}
+                    @{ string ScanMask = "Default"; }
+                    @{ string PortRange = "Default";}
+                    @{ string MaxNetworkProbes = "Default";}
+                    @{ string MaxPortProbes = "Default";}
+                    @{ string NetworkProbeTimeoutMs = "Default";}
+                    @{ string PortProbeTimeoutMs = "Default";}
+                    @{ string IdleTimeBetweenScansSec = "Default";}
```

Comment:
I think it would be nice to specify what the default value is where possible. 
E.g: IdleTimeBetweenScansSec = "300s(default)"

Reply:
Partially fixed by the other PR currently open